R/make-v-caption.R

```diff
@@ -5,12 +5,12 @@
 #'
 #' @keywords internal
 make_v_caption <- function(.df, .subject_col, font_size){
-  style <- glue('caption-side: top; text-align: left; color:#0053a0; font-size:{font_size}pt;
+  style <- glue('caption-side: top; text-align: left; color:#FFFFFF; background-color:#5A5A5A; font-size:{font_size}pt;
                 font-weight: bold; position: fixed;left: 12em; top: 1em; padding: 3px')
 
   cap_txt <- if(!is.null(.subject_col)){
     num_subj <- .df %>% dplyr::count(!!!syms(.subject_col)) %>% nrow()
-    glue('Number of Subjects: {num_subj}')
+    glue('N Subjects ({.subject_col}): {num_subj}')
   }else{
     'No subjects detected'
```

Comment:
I just realized I didnt add a `"."` to the `font_size` argument (i.e. `.font_size`). Up to you if you want to fix that, given that it's an internal function

Reply:
Not sure what's going on, but im getting an error when trying to use one of my test datasets from before:

```r
> devtools::load_all(); v(tmp)
ℹ Loading mrgda
Error in writeImpl(text) :
Text to be written must be a length-one character vector
```

Reply:
@barrettk Fixed in final commit